config/config_params.go

```diff
@@ -242,8 +242,8 @@ type Config struct {
 	PrometheusGoMetricsEnabled      bool   `config:"bool;true"`
 	PrometheusProcessMetricsEnabled bool   `config:"bool;true"`
 
-	FailsafeInboundHostPorts  []ProtoPort `config:"port-list;tcp:22,udp:68,tcp:179,tcp:2379,tcp:2380,tcp:5473,tcp:6443,tcp:6666,tcp:6667;die-on-fail"`
-	FailsafeOutboundHostPorts []ProtoPort `config:"port-list;udp:53,udp:67,tcp:179,tcp:2379,tcp:2380,tcp:5473,tcp:6443,tcp:6666,tcp:6667;die-on-fail"`
+	FailsafeInboundHostPorts  []ProtoPort `config:"port-list;0.0.0.0/0:tcp:22,0.0.0.0/0:udp:68,0.0.0.0/0:tcp:179,0.0.0.0/0:tcp:2379,0.0.0.0/0:tcp:2380,0.0.0.0/0:tcp:5473,0.0.0.0/0:tcp:6443,0.0.0.0/0:tcp:6666,0.0.0.0/0:tcp:6667;die-on-fail"`
```

Comment:
What do you think about changing the ordering to be `<protocol>:<cidr>:<port>`? I think it matches the most with what we previously had so that it doesn't change too much for our users.

Reply:
We could do, the reason why I did this format was that it made it backwards compatible with the current structure of `<protocol>:<port>`.

See: https://github.com/projectcalico/felix/pull/2646/files#diff-2e8a8c60cee2386f6eb73e92222a9ba3ebb38d34e53edbed3b8fcb6d259c0214L317

If we switch to `<protocol>:<cidr>:<port>` it will be a breaking change. I do however think this structure is more syntactically correct and am happy to change the ordering if this is preferred.

Reply:
I don't think it's a breaking change if we implement it correctly - the field field could just support two formats:

- <protocol>:<port>
- <protocol>:<cidr>:<port>

There's no ambiguity there, because if three segements are given then it is clearly the latter, and if 2 segments are given then it must be the "old" style.

Does that seem right?

Reply:
Ah yes, you're right. Seems I was having a brain-fart moment when replying. Will switch to the new structure.